repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -462,6 +468,20 @@ export const completeMultipleProducts = (
     continueButtonClick();
 };
 
+export const clickRandomElementsInTable = (tableName: string, elementId: string): void => {
+    getElementByClass(tableName)
+        .find('tr')
+        .then((elm) => {
+            const numberOfServices = elm.length;
+            for (let i = 0; i < 3; i++) {
+                const randomSelector = getRandomNumber(0, numberOfServices - 1);
+                elm.get(randomSelector).setAttribute('selected', 'selected');
```

Comment:
What are we seeking to achieve by clicking on the services?

Reply:
So this asserts that we can navigate to the pointToPointProducts page for a random set of services, and to the productDetails page for a random set of 'other products', depending on what's passed to it. I have removed line 478 though, as I don't think it was doing anything!